src/styles/layout.css

```diff
@@ -118,6 +118,15 @@ header .header-social li a:hover {
 
 /* primary navigation
 --------------------------------------------------------------------- */
+
+.NavLink {
```

Comment:
Was this not done and merged already?

Reply:
No this is done to show the get started page now in the navigation bar instead of the home page sections

Reply:
OK